src/fuse.rs

```diff
@@ -552,24 +541,27 @@ mod sys {
     }
 }
 
-
 #[cfg(feature="rust-mount")]
-fn fuse_mount_sys(mountpoint: &PathBuf, flags: u64, mnt_opts: &Vec<String>) -> i32
+fn fuse_mount_sys(mountpoint: &PathBuf, flags: u64, mnt_opts: &mut FuseOpts) -> i32
```

Comment:
I'm not sure I like that FuseOpt is using `mut`. Is there a reason to it besides "i'm using add_opt as a way to get defaults" ? It's a nitpick, not blocking.

Reply:
done 958c8b3